src/Microsoft.OData.Client/HttpStack.cs

```diff
@@ -8,27 +8,12 @@ namespace Microsoft.OData.Client
 {
     /// <summary>Represents the type of HTTP implementation to use when accessing the data service.Supported only by the WCF Data Services 5.0 client for Silverlight.</summary>
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Justification = "Accurately describes what the enum represents, and also this class has already shipped as a public type.")]
-#if !ASTORIA_LIGHT
     internal enum HttpStack

```

Comment:
remove this enum will bring a tons of code changes. we'd better do it in another task.